.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
```

Comment:
Jaco has this eslint & tslint config that he uses in all of the PolkadotJS packages and we're also using it in quite a few of the Parity JS projects, so I'd say it's battle tested.
Might be worth having a look even if you don't want to 100% copy it:
https://github.com/polkadot-js/dev/blob/master/packages/dev/config/eslint.js
https://github.com/polkadot-js/dev/blob/master/packages/dev/config/tsconfig.json

Reply:
Added codestyle change as a part of #35